SEED-platform/seed
# upload media to s3

#### Any background context you want to provide?
When we upload files, we want to send them to s3 to limit storing locally

#### What's this PR do?

send uploaded files to an s3 bucket

#### How should this be manually tested?
first make sure you have no docker containers, images or volumes, clear everything.
sudo docker volume create seed_media
sudo docker volume create seed_pgdata
sudo docker-compose up db-postgres db-redis
sudo docker-compose up --build web web-celery
#### What are the relevant tickets?
#3446 
#### Screenshots (if appropriate)
